packages/jest-cli/src/__tests__/TestRunner-test.js

```diff
@@ -38,6 +38,115 @@ test('.addReporter() .removeReporter()', () => {
   expect(runner._dispatcher._reporters).not.toContain(reporter);
 });
 
+describe('noDefaultReporters option', () => {
+  let runner;
+  
+  test('does not add reporters if true', () => {
+    runner = new TestRunner({}, {noDefaultReporters: true}, {});
+    expect(runner._dispatcher._reporters).toHaveLength(0);
+  });
+
+  test('adds reporters if undefined', () => {
+    runner = new TestRunner({}, {}, {});
+    expect(runner._dispatcher._reporters).toHaveLength(2);
+  });
+
+  test('adds reporters if false', () => {
+    runner = new TestRunner({}, {noDefaultReporters: false}, {});
+    expect(runner._dispatcher._reporters).toHaveLength(2);
+  });
+});
+
+// Checking for the method _addCustomReporters
+// Custom reporters used here are the reporters within the package
+// No extra reporters are included to be used
+describe('_addCustomReporters', () => {
```

Comment:
@dmitriiabramov Thanks a lot that works, based on this I have removed additional tests I added for private method `_addCustomReporters` and will now add integration tests for custom_reporters separately.